src/enclave/ecalls.cpp

```diff
@@ -9,21 +9,24 @@
 #include "crypto/encryption_engine.h"
 #include "ortoa_t.h"
 
-void access_data(int op_const, const char* in_val, size_t in_size, const char* update_val, size_t update_size, unsigned char* cipher_text, size_t* out_size) {
+void access_data(int op_const, const char *in_val, size_t in_size,
```

Comment:
I think this file got formatted, no changes were made I believe

Reply:
You're right, no changes were made